lib/rules/no-spreading-accumulators.js

```diff
@@ -12,13 +12,14 @@ module.exports = {
     type: 'problem',
     fixable: 'code',
     schema: [],
+    messages: {
```

Comment:
This fixes the one existing lint error.

https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/HEAD/docs/rules/prefer-message-ids.md